package.json

```diff
@@ -21,6 +21,7 @@
   "dependencies": {
     "html-minifier": "^4.0.0",
     "http-link-header": "^1.0.2",
+    "rollup-dependency-tree": "0.0.10",
```

Comment:
Is there a reason for this to be an external dependency? Not that it matters _a lot_, since it's only going to be in play when building the Sapper app, but, especially since this is already pinned to a particular version, would it make sense to bundle this instead?

Reply:
It could be useful to other projects, which was part of it. But the main reason I did it was that I was having a very difficult time getting the test infrastructure setup within Sapper without breaking the existing tests. The library has some rollup output saved as JSON that I then load and test that it's handled correctly. Sapper's test setup really didn't like that I was trying to import JSON as a module and I couldn't find any combination of settings that would allow me to do that without breaking the setup of existing tests